### ISSUE TITLE
The web data connector created by your generator does not work in 10.1 or 10.2

### ISSUE THREAD

**Issue by idrinks** [tableau-mkt/generator-web-data-connector#58]
When you deploy the web data connector created by your web data generator you get the following error ReferenceError: Can't find variable: Promise.  What do I need to do to get past this error.  Please note that the web data connector does work when I test it with the simulator.

**Comment by iamEAP** [comment]
Hi @idrinks! Thanks for the note.  Sounds like the issue could be resolved by adding a Promise polyfill to the WDC.  You could either include a script tag like the following in your markup, or include a local file with the same contents:

```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/es6-promise/4.1.0/es6-promise.min.js"></script>
```

**Comment by iamEAP** [comment]
I'll have to dig into the generator to see why a polyfill isn't already being included.

**Comment by iamEAP** [comment]
@idrinks It's possible the issue you were seeing is a result of something fixed in `v2.0.0-beta.2` (which you can update to by running `npm install -g generator-web-data-connector`).

Let me know if you're still experiencing issues with this; for now I'm closing this issue.
